presto-main/src/main/java/com/facebook/presto/spiller/SingleStreamSpiller.java

```diff
@@ -57,6 +57,12 @@
      */
     ListenableFuture<List<Page>> getAllSpilledPages();
 
+    /**
+     * Commit the spill file. Once committed, the spill file can no longer be modified
```

Comment:
Maybe also update the comments on `getSpilledPages` and `getAllSpilledPages` mentioning that the spiller will commit automatically if not commited.